dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -769,6 +769,10 @@ class PerfTestWithSkSL extends PerfTest {
       _flutterPath,
       <String>[
         'run',
+        if (deviceOperatingSystem == DeviceOperatingSystem.ios)
```

Comment:
This also needs to apply to the startup test and whatever the SkSL test is

Reply:
Done.